bitnami/keycloak/README.md

```diff
@@ -0,0 +1,370 @@
+| `command`                               | Override default container command (useful when using custom images)                     | `nil`                                                   |
+| `args`                                  | Override default container args (useful when using custom images)                        | `nil`                                                   |
+| `extraEnvVars`                          | Extra environment variables to be set on Keycloak container                              | `{}`                                                    |
+| `extraEnvVarsCM`                        | Name of existing ConfigMap containing extra env vars                                     | `nil`                                                   |
+| `extraEnvVarsSecret`                    | Name of existing Secret containing extra env vars                                        | `nil`                                                   |
+
+### Keycloak deployment/statefulset parameters
+
+| Parameter                             | Description                                                                                | Default                                                 |
+|---------------------------------------|--------------------------------------------------------------------------------------------|---------------------------------------------------------|
+| `replicaCount`                        | Number of Keycloak replicas to deploy                                                      | `1`                                                     |
+| `containerPorts.http`                 | HTTP port to expose at container level                                                     | `8080`                                                  |
+| `containerPorts.https`                | HTTPS port to expose at container level                                                    | `8443`                                                  |
+| `podSecurityContext`                  | Keycloak pods' Security Context                                                            | Check `values.yaml` file                                |
+| `containerSecurityContext`            | Keycloak containers' Security Context                                                      | Check `values.yaml` file                                |
+| `resources.limits`                    | The resources limits for the Keycloak container                                            | `{}`                                                    |
+| `resources.requests`                  | The requested resources for the Keycloak container                                         | `{}`                                                    |
+| `lifecycleHooks`                      | LifecycleHooks to set additional configuration at startup.                                 | `{}` (evaluated as a template)                          |
```

Comment:
Duplicated parameter. Following `values.yaml` order, I think this is the line that should be deleted.